client/state/plugins/installed/test/plugins.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable max-len */
```

Comment:
If you have test data used for mocks, they should be in a subdirectory within `test` because of the way test discovery works (i.e. as-is, Mocha will try to process this file as a test file).

See: https://github.com/Automattic/wp-calypso/tree/master/test#how-to-add-a-new-test-file